src/archivist/fetcher/htmlOnlyFetcher.js

```diff
@@ -1,6 +1,6 @@
+import convertBody from '@opentermsarchive/fetch-charset-detection'; // eslint-disable-line import/no-unresolved
```

Comment:
Do we still need the no-unresolved?

Reply:
I guess, there are no changes on that side.